log/logger_stdout.go

```diff
@@ -0,0 +1,35 @@
+package log
+
+import (
+	"github.com/op/go-logging"
+	"os"
+)
+
+// StdoutLogger outputs logs to standard output.
+type StdoutLogger struct {
+	basicLogger
+}
+
```

Comment:
You could add a line here that would ensure in compile time already that the logger implementation complies with your interface:

```go
var _ BaseLogger = (*StdoutLogger)(nil)
```

If you decide to do so, please do it for all other logger implementations as well.